Shear/App.swift

```diff
@@ -13,13 +13,21 @@ class App {
     let statusMenu = StatusMenu()
     
     let history: HistoryType
-    
+
+
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/c5a4410df1234342937e628f80ec930f9115845a/Shear/App.swift#L20"></span>Limit vertical whitespace to a single empty line. Currently 5.<br />
<code>vertical_whitespace</code> <code>App.swift:20</code></del></td>
    </tr>
  </tbody>
</table>